backend/main.py

```diff
@@ -148,66 +148,79 @@ def get_file_info():
     url = request.args.get('originUrl', default=None, type=str)
     strategy = request.args.get('strategy', default=None, type=str)
     number = request.args.get('number', default=None, type=str)
-    inactiveChannels = request.args.get('inactiveChannels', default=None, type=str)
-    startTime = request.args.get('startTime', default=None, type=str)
-    endTime = request.args.get('endTime', default=None, type=str)
+    inactive_channels = request.args.get(
+        'inactiveChannels', default=None, type=str)
+    start_time = request.args.get('startTime', default=None, type=str)
+    end_time = request.args.get('endTime', default=None, type=str)
     if url:
-      return redirect(url+'&strategy='+strategy+"&number="+number+"&inactiveChannels="+inactiveChannels+"&startTime="+startTime+"&endTime="+endTime, code=302)
-    else:
-      return redirect('https://tank-big-data-plotting-285623.googleplex.com/', 302)
+        return redirect(
+            url + '&strategy=' + strategy + '&number=' + number +
+            '&inactiveChannels=' + inactive_channels + '&startTime=' +
+            start_time + '&endTime=' + end_time,
+            code=302)
+    return redirect('https://tank-big-data-plotting-285623.googleplex.com/',
+                    302)
 
 
 def make_response(response_body):
     response = app.make_response(response_body)
     response.headers['Access-Control-Allow-Credentials'] = 'true'
-    # response.headers['Access-Control-Allow-Origin'] = 'https://api-dot-tank-big-data-plotting-285623.googleplex.com'
+    # response.headers['Access-Control-Allow-Origin'] =
+    # 'https://api-dot-tank-big-data-plotting-285623.googleplex.com'
     return response
-    
+
+
 @app.route('/downsample')
-def scanFiles():
-  client = storage.Client()
-  raw_bucket = client.bucket(RAW_BUCKET)
-  preprocess_bucket = client.bucket(PREPROCESS_BUCKET)
-  blobs = client.list_blobs(RAW_BUCKET)
-
-  names = [blob.name for blob in blobs]
-  files_preprocess = [MultipleLevelPreprocess(
-      name, PREPROCESS_DIR, preprocess_bucket, raw_bucket) for name in names]
-
-  response_data = list()
-  for preprocess, name in zip(files_preprocess, names):
-      if not preprocess.is_preprocessed():
-          print("Downsampling file "+name)
-          number_per_slice = NUMBER_OF_RECORDS_PER_SLICE
-          downsample_factor = DOWNSAMPLE_LEVEL_FACTOR
-          minimum_number_level = MINIMUM_NUMBER_OF_RECORDS_LEVEL
-          client = storage.Client()
-          preprocess = MultipleLevelPreprocess(name, PREPROCESS_DIR, client.bucket(
-              PREPROCESS_BUCKET), client.bucket(RAW_BUCKET))
-          error = preprocess.preprocess(
-              number_per_slice, downsample_factor, minimum_number_level)
-
-          if error is not None:
-              response = make_response(error)
-              return response
-
-  print("Downsample scan complete")
-  response = make_response('Downsample scan complete')
-  return response
+def scan_files():
```

Comment:
Do you know if this is a safe change (for the linter)? I couldn't find scanFiles being referenced anywhere in the project, so I'm assuming the '/downsample' part is the important one.

Reply:
It should be good